catalog/precommit/dagger.go

```diff
@@ -5,17 +5,26 @@ package precommit
 
 import (
 	"context"
+	"embed"
+	"fmt"
+	"io"
+
+	"dagger.io/dagger"
 
 	"github.com/mesosphere/daggers/daggers"
 	"github.com/mesosphere/daggers/daggers/containers"
 )
 
 const (
-	configFileName      = ".pre-commit-config.yaml"
+	configFileName      = "pre-commit-config.yaml"
```

Comment:
💭 should we consider making this config file path configurable? I'm thinking some of the pre-commit config would be repository specific.

Reply:
Let me make an issue for that!

Reply:
sounds good! thanks